packages/eas-json/schema/eas.schema.json

```diff
@@ -168,6 +168,14 @@
           "markdownDescription": "When enabled, for iOS, bumps the last component of `expo.ios.buildNumber` (e.g. `1.2.3.39` -> `1.2.3.40`) and for Android, bumps `expo.android.versionCode` (e.g. `3` -> `4`).",
           "default": false
         },
+        "buildArtifactsPaths": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
```

Comment:
@byCedric is this the correct way to define an array of strings? How can I test it?

Reply:
> is this the correct way to define an array of strings?
yes, it works for mee

Not sure how to test it in vscode, I'm testing that by changing URL to the schema in my vim config and opening eas.json.

You could probably use sth like this https://www.jsonschemavalidator.net/